draft-ietf-quic-transport.md

```diff
@@ -420,8 +361,9 @@ conditions for opening a stream are slightly more complex for a bidirectional
 stream because the opening of either send or receive sides causes the stream
 to open in both directions.
 
```

Comment:
I'm not sure how to formulate this, but there is a difference between the endpoints and view, including the state machine, and the overall distributed view: A stream can only be opened by the initiator, but the first materialized view of a frame can be receiver side sending a packet to a bidi stream. This does not mean that the receiver is opening the stream, but to the receiver it looks like the stream is being opened (although it already knows that is is open from receiving a higher valued Stream ID).

In short, I think it is wrong to state that receive side can open a stream, but the correct formulation is not clear to me.

Reply:
Fair point, though as you note, it's not clear exactly what to replace this with. Feel free to suggest a PR for it if you come up with something here.